From ebe7a502fb37ce2c684a312e68195e097cf82cc8 Mon Sep 17 00:00:00 2001
From: Wenyou Yang <wenyou.yang@atmel.com>
Date: Thu, 11 Dec 2014 08:59:18 +0800
Subject: [PATCH] fix the wrong defines for the ZQ field of MPDDRC_CR in the
 include/arch/at91_ddrsdrc.h.

They were codeed as if they were binary, but they are in hex.

This code was introduced on 2013-11-28, Commit: "driver: ddramc: add driver to support the low_power DDR2-SDRAM device",

Reported by rwille from the github: linux4sam/at91bootstrap

Signed-off-by: Wenyou Yang <wenyou.yang@atmel.com>
---
 include/arch/at91_ddrsdrc.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/arch/at91_ddrsdrc.h b/include/arch/at91_ddrsdrc.h
index 3f19a1f..31ec3c0 100644
--- a/include/arch/at91_ddrsdrc.h
+++ b/include/arch/at91_ddrsdrc.h
@@ -102,10 +102,10 @@
 #define 	AT91C_DDRC2_DIS_DLL_DISABLED	(0x0UL << 9)
 #define 	AT91C_DDRC2_DIS_DLL_ENABLED	(0x1UL << 9)
 #define AT91C_DDRC2_ZQ		(0x03 << 10)
-#define		AT91C_DDRC2_ZQ_INIT		(0x00 << 10)
-#define		AT91C_DDRC2_ZQ_LONG		(0x01 << 10)
-#define		AT91C_DDRC2_ZQ_SHORT		(0x10 << 10)
-#define		AT91C_DDRC2_ZQ_RESET		(0x11 << 10)
+#define		AT91C_DDRC2_ZQ_INIT		(0x0 << 10)
+#define		AT91C_DDRC2_ZQ_LONG		(0x1 << 10)
+#define		AT91C_DDRC2_ZQ_SHORT		(0x2 << 10)
+#define		AT91C_DDRC2_ZQ_RESET		(0x3 << 10)
 #define AT91C_DDRC2_OCD		(0x7UL << 12)
 #define 	AT91C_DDRC2_OCD_EXIT		(0x0UL << 12)
 #define 	AT91C_DDRC2_OCD_DEFAULT		(0x7UL << 12)
-- 
1.9.1

